### PR TITLE
[FLINK-33235][doc] Update OLAP QuickStart doc and translate to Chinese

### PR DESCRIPTION
**Pull request by xiangyuf**
## What is the purpose of the change

Update Flink OLAP quickstart and translate to Chinese

## Brief change log

- Follow the [documentation style guide](https://flink.apache.org/how-to-contribute/documentation-style-guide/) use 'active voice' and 'you' instead of 'passive voice' and 'we' in the doc
- Add architecture diagram in the doc
- Fix typos and misconfigured options in the doc
- Add new introduced options
- Translate to Chinese

## Verifying this change

This change is a trivial rework / code cleanup without any test coverage.

## Does this pull request potentially affect one of the following parts:

  - Dependencies (does it add or upgrade a dependency): no
  - The public API, i.e., is any changed class annotated with `@Public(Evolving)`: no
  - The serializers: no
  - The runtime per-record code paths (performance sensitive): no
  - Anything that affects deployment or recovery: JobManager (and its components), Checkpointing, Kubernetes/Yarn, ZooKeeper: no
  - The S3 file system connector: no

## Documentation

  - Does this pull request introduce a new feature? no

### CODE REVIEW

**Comment by xiangyuf** [comment]
@flinkbot run azure

**Comment by xiangyuf** [comment]
@flinkbot run azure

**Comment by xiangyuf** [comment]
@flinkbot run azure

**Comment by xiangyuf** [comment]
@flinkbot run azure

**Comment by xiangyuf** [comment]
@KarmaGYZ @FangYongs Hi, can you help review this when you have time?

**Comment by xiangyuf** [comment]
@flinkbot run azure

File: docs/content.zh/docs/dev/table/olap_quickstart.md [hunk 1]
```diff
@@ -102,98 +110,98 @@ GROUP BY  buyer
...
+这个章节会向你介绍一些在生产环境中使用 Flink OLAP 服务的建议。
 
-## Cluster Deployment
+## 客户端
 
-In production, we recommend to use Flink Session Cluster, Flink SQL Gateway and Flink JDBC Driver to build an OLAP service.
+### Flink JDBC Driver
 
-### Session Cluster
+Flink JDBC Driver 提供了底层的连接管理能力，方便用户使用并向 SQL Gateway 提交查询请求。在实际的生产使用中，用户需要注意如何复用 JDBC 连接，来避免 Gateway 频繁的执行 Session 相关的创建及关闭操作，从而减少端到端的作业耗时。详细信息可以参考文档 [Flink JDBC Driver]({{{<ref "docs/dev/table/jdbcDriver">}}})。
```
Comment (KarmaGYZ) [inline docs/content.zh/docs/dev/table/olap_quickstart.md:123]: broken link.

Comment (KarmaGYZ) [reply docs/content.zh/docs/dev/table/olap_quickstart.md:123]: broken in english version.

File: docs/content.zh/docs/dev/table/olap_quickstart.md [hunk 2]
```diff
@@ -24,64 +24,72 @@ specific language governing permissions and limitations
...
-## Advantage
+Flink OLAP 服务整体由3个部分组成，包括：客户端，Flink SQL Gateway 和 Flink Session Cluster。
 
-* **Massively Parallel Processing**
-  * Flink OLAP runs naturally as an MPP(Massively Parallel Processing) system, which supports low-latency ad-hoc queries
-* **Reuse Connectors**
-  * Flink OLAP can reuse rich connectors in Flink ecosystem.
-* **Unified Engine**
-  * Unified computing engine for Streaming/Batch/OLAP.
+* **客户端**: 可以是任何可以和 Flink SQL Gateway 交互的客户端，包括：SQL Client，Flink JDBC Driver 等等；
```
Comment (KarmaGYZ) [inline docs/content.zh/docs/dev/table/olap_quickstart.md:38]: Add links for them.

File: docs/content.zh/docs/dev/table/olap_quickstart.md [hunk 3]
```diff
@@ -24,64 +24,72 @@ specific language governing permissions and limitations
...
+* **Flink SQL Gateway**: Flink SQL Gateway 服务主要用作 SQL 解析、元数据获取、统计信息分析、Plan 优化和集群作业提交；
```
Comment (KarmaGYZ) [inline docs/content.zh/docs/dev/table/olap_quickstart.md:39]: ditto.

File: docs/content.zh/docs/dev/table/olap_quickstart.md [hunk 4]
```diff
@@ -24,64 +24,72 @@ specific language governing permissions and limitations
...
+* **Flink Session Cluster**: OLAP 查询建议运行在 Session 集群上，主要是可以减少集群启动时的额外开销；
```
Comment (KarmaGYZ) [inline docs/content.zh/docs/dev/table/olap_quickstart.md:40]: ditto for session cluster.

File: docs/content/docs/dev/table/olap_quickstart.md [hunk 5]
```diff
@@ -24,32 +24,39 @@ specific language governing permissions and limitations
...
+* **Flink SQL Gateway**: The SQL Gateway provides an easy way to parse the sql query, look up the metadata, analyze table stats, optimize the plan and submit JobGraphs to cluster.
+* **Flink Session Cluster**: OLAP queries run on session cluster, mainly to avoid the overhead of cluster startup.
+
+{{< img src="/fig/olap-architecture.svg" alt="Illustration of Flink OLAP Architecture" width="85%" >}}
 
 ## Advantage
 
 * **Massively Parallel Processing**
-  * Flink OLAP runs naturally as an MPP(Massively Parallel Processing) system, which supports low-latency ad-hoc queries
+  * Flink OLAP runs naturally as an MPP(Massively Parallel Processing) system, which enables planners to easily adjust the job parallelism to fulfill queries' latency requirement under different data sizes.
```
Comment (KarmaGYZ) [inline docs/content/docs/dev/table/olap_quickstart.md:45]: Remove the MPP phrase.

File: docs/content.zh/docs/dev/table/olap_quickstart.md [hunk 6]
```diff
@@ -24,64 +24,72 @@ specific language governing permissions and limitations
...
+  * Flink OLAP 可以复用 Flink 生态中丰富的 [连接器]({{< ref "docs/connectors/table/overview.md" >}})
+* **统一引擎**
+  * 支持流 / 批 / OLAP 的统一计算引擎
+
+# 本地运行
+本章将指导用户如何在本地试用 Flink OLAP 服务。
+
+## 下载 Flink
+
+这里的方法和[本地安装]({{< ref "docs/try-flink/local_installation" >}})中记录的步骤类似。Flink 可以运行在任何类 UNIX 的操作系统下面, 例如：Linux, Mac OS X 和 Cygwin (for Windows)。你需要在本地安装好 __Java 11__，可以通过下述命令行的方式检查安装好的 Java 版本：
```
Comment (KarmaGYZ) [inline docs/content.zh/docs/dev/table/olap_quickstart.md:60]: We recommend user to use JDK 17 below.

**Review by KarmaGYZ — CHANGES_REQUESTED** [review]

**Review by KarmaGYZ — APPROVED** [review]

**Pull request merged**